docs/integrations/pack-docs.md

```diff
@@ -55,3 +73,195 @@ It is possible to add an image placeholder which links to an external video. For
+
+```demisto-sdk find-dependencies -p 'PackName'``` - if you don’t have the id_set.json file
+
+*The `id_set.json` file defines the relationships between content elements, and dictates the dependencies accordingly. If you don't have the `id_set.json` file, it will be generated automatically during the `find-dependencies` phase. It will scan your local copy of the repository. This process may take time. If you did not make any changes that could affect dependencies in your branch, you may provide your existing `id_set.json` file to save time.*
+
+
+### What are some examples of dependencies?
+- A playbook from QRadar pack uses a playbook from AccessInvestigation pack
+- A playbook from Employee Offboarding pack uses an automation from Impossible Traveler pack
+- A classifier from EWS pack uses incident fields from Phishing pack
+
+The dependency logic is as follows:
+```
+integration <-> mapper, incident type, indicator type, indicator field
+classifier <-> incident type, integration 
+mapper <-> incident type, incident field
+incident type <-> playbook, script, integration
+layout <-> incident type, indicator type, incident field, indicator field
+incident field <-> incident type
+playbook <-> incident field, indicator field, script, integrations
+Widget <-> script
+```
+
+### What should I do about the dependencies?
+When a dependency is required, that means that in order to use a certain pack, the user MUST install a different pack - a behavior we want to eliminate as much as possible. 
+
+Sometimes the dependency is justified. In this case Gmail depends on Phishing. Phishing contains content that is absolutely needed for Gmail, so we can avoid creating duplicate content in both packs, and instead we will acknowledge this as a mandatory dependency. Due to the fact that Phishing contains such core elements, we've also decided to make Phishing a Core pack, which ships out-of-the-box, so this causes no issue and mandatory can stay true:
+![image](https://user-images.githubusercontent.com/43602124/89119495-901aca80-d4b7-11ea-999e-100d8d58663e.png)
+
+In other cases the dependency must be fixed by us, manually. 
+
+### How do I fix the dependencies?
+The cases can vary from one dependency to another. Fixing a dependency involves 3 parts:
+1. Making the necessary adaptation in the content (changing playbook, merging packs, moving files to another pack, replacing deprecated script with newer script)
+2. Manually changing “mandatory” to “false” in the pack dependencies - only if the dependency really is optional and not mandatory, for example if an integration is used after a condition that ensures that it's enabled, and the flow continues normally otherwise. Another possible optional dependency is when an automation or a subplaybook is configured to be skipped if the pack is unavailable (through the advanced task settings).
+3. Removing the displayedImages section
```

Comment:
What does this mean?

Reply:
@michalgold why are we removing the section? Was it some bug?